iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -1381,6 +1381,11 @@ public void setTableParametersForCTLT(org.apache.hadoop.hive.ql.metadata.Table t
       SessionStateUtil.addResourceOrThrow(conf, hive_metastoreConstants.PARTITION_TRANSFORM_SPEC, spec);
       tbl.getSd().getCols().addAll(tbl.getPartitionKeys());
       tbl.getTTable().setPartitionKeysIsSet(false);
+
+      // Remove any transactional properties as they aren't relevant in case of iceberg tables.
+      tbl.getParameters().keySet().removeIf(
+          key -> hive_metastoreConstants.TABLE_IS_TRANSACTIONAL.equalsIgnoreCase(key) ||
+              hive_metastoreConstants.TABLE_TRANSACTIONAL_PROPERTIES.equalsIgnoreCase(key));
```

Comment:
should we filter out `hive_metastoreConstants.TABLE_TRANSACTIONAL_PROPERTIES` as well?